.travis.yml

```diff
@@ -39,23 +40,27 @@ env:
     - PERL5LIB=~/perl5/lib/perl5/x86_64-linux-gnu-thread-multi:~/perl5/lib/perl5:/etc/perl:/usr/local/lib/perl/5.14.2:/usr/local/share/perl/5.14.2:/usr/lib/perl5:/usr/share/perl5:/usr/lib/perl/5.14:/usr/share/perl/5.14:/usr/local/lib/site_perl
     - USE_OMNIBUS_FILES=0
     - CHEFDK_GECODE_PATH=/opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/dep-selector-libgecode-1.0.2/lib/dep-selector-libgecode/vendored-gecode
-
-#    - LUALIB=~/.luarocks/lib/lua/5.2
-  matrix:
```

Comment:
Is removing the matrix build strictly necessary?  I'm concerned that (1) it will slow down the build (note that if we are just comparing build time on the reported total, the reported total is incorrect when using matrix builds as it sums all the builds even when they happen in parallel), and (2) that the feedback we get from builds will be much less clear.  Right now it is very easy to see which component failed its tests.